resources/ext/config/conf.d/puppetserver.conf

```diff
@@ -2,7 +2,7 @@
 jruby-puppet: {
     # Where the puppet-agent dependency places puppet, facter, etc...
     # Puppet server expects to load Puppet from this location
-    ruby-load-path: [/opt/puppetlabs/puppet/lib/ruby/vendor_ruby]
+    ruby-load-path: [/opt/puppetlabs/puppet/lib/ruby/vendor_ruby, /opt/puppetlabs/puppet/cache/lib]
```

Comment:
@joshcooper Thanks for following up on this.  @cprice404 raised an interesting question in the comments [SERVER-571](https://tickets.puppetlabs.com/browse/SERVER-571), "why isn't 1.0.8 affected?"

I assumed that puppetserver 1.0.8 would not suffer this issue because 1.0.x does not have the AIO pathing changes.  However, upon verification of that assumption I just found out 1.0.8 _is_ suffering from this issue.

Even more perplexing to me is that puppetserver 1.0.2 is _not_ experiencing this issue.

However, both 1.0.2 and 1.0.8 have the same ruby-load-path setting of `ruby-load-path: [/usr/share/ruby/vendor_ruby/]`.

I'll post more details in that ticket, but at this point I'm sufficient confused about the nature of this issue that I'm also concerned this pull request isn't the correct solution.